src/design-system/box/box.component.tsx

```diff
@@ -33,6 +34,7 @@ export type Props = PropsWithChildren<BoxProps> &
 export const Box = forwardRef<HTMLDivElement | null, Readonly<Props>>(
   (
     {
+      background,
```

Comment:
I'll get rid of this as we can use `Card.Elevated` for the same purpose instead of the Box.